### ISSUE TITLE
Run scripts cannot handle spaces in folder names

### ISSUE THREAD

**Issue by fred-dev** [openframeworks/openFrameworks#6372]
The scripts for downloading libs fail if there are spaces in the fodler names. I have a use case where I cannot control the folder names, It would be great if the scripts could handle spaces.

**Comment by bakercp** [comment]
This goes for apothecary too.

**Comment by ofTheo** [comment]
Are you able to tell which lines this fails on? 
I took a quick look and didn't see anything that stood out.

**Comment by fred-dev** [comment]
For download_libs.sh for OSX and IOS I get an error on  line 141. I don't get further than that.

**Comment by kkshmz** [comment]
Hi, I found that double quoting works quite nicely on OSX, I can do pull request if you want to.
cd "$SCRIPT_DIR"  on line 141

with that being said, I don't know how this would work on other platforms as you could potentially pipe a sed arguement to escape the spaces. 
`SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd | sed 's/ /\\ /g' )"`

**Comment by arturoc** [comment]
yes adding quotes should solve the problem in every platform in any case the script is tested in the CI servers so if you want to send a PR we'll see if at least it keeps working fine on every platform

**Comment by fred-dev** [comment]
Double quotes were added in the scripts/dev/download_libs.sh script, but for this to work on OSX they also need to be added to the scripts/ios/download_libs.sh and scripts/osx/download_libs.sh scripts.

**Comment by danomatika** [comment]
Yes. Path variables should always be escaped in the apothecary scripts. Hell, for shell scripts in general.
